content/info/software.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Software development"
+date = 2024-01-13
++++
+
+## Contract software development
```

Comment:
maybe instead
Software development and consulting

Reply:
```suggestion
## Software development and consulting
```